HandsOn/upgrade/KF_upgrade_1_2_to_1_3.md

```diff
@@ -0,0 +1,321 @@
+# Kubeflow upgrade 1.2 to 1.3 (multi-user, AppID, IKS, K8s v1.19)
+_Raw Technical Writeup_
+
+*Credits to Tommy Li and Mofi Rahman!*
+
+Prerequisites:
+- bash
+- kubectl, logged in to cluster
+- old KF 1.2 installation
```

Comment:
FYI, 1.2 single user have different database schema for the runs and experiments. Although the upgrade process will be similar, we need to apply the namespace column from #5 to all the db tables.
```suggestion
- old KF 1.2 multi-user installation
```
```suggestion
- old KF 1.2 installation
```